README.rst

```diff
@@ -23,6 +23,13 @@ Using PIP via PyPI
 
     pip install fuzzywuzzy
 
+or
+
+.. code:: bash
+
+    pip install fuzzywuzzy[speedup]
```

Comment:
Can you add a `#` comment in this codeblock that explains what this does differently?